project/Builder.scala

```diff
@@ -46,6 +46,7 @@ object Builder {
       "commons-io"           % "commons-io"      % "2.6",
       "javax.inject"         % "javax.inject"    % "1",
       "ch.qos.logback"       % "logback-classic" % "1.2.3",
+      "org.codehaus.janino"  % "janino"          % "3.0.11",
```

Comment:
Is this used anywhere?

Reply:
@neongreen No it is not. But it needs for using `if condition` profiles with logback.